helm-charts/chatqna/templates/deployment.yaml

```diff
@@ -39,20 +39,28 @@ spec:
               {{- if eq .Values.CHATQNA_TYPE "CHATQNA_FAQGEN"}}
               value: {{ .Release.Name }}-llm-uservice
               {{- else }}
+              {{- if .Values.ollama.enabled }}
+              value: {{ .Release.Name }}-ollama
+              {{- else }}
               {{- if .Values.vllm.enabled }}
               value: {{ .Release.Name }}-vllm
               {{- else }}
               value: {{ .Release.Name }}-tgi
               {{- end }}
               {{- end }}
+              {{- end }}
```

Comment:
IMHO using `else if` would make this clearer than nested `if`s: https://helm.sh/docs/chart_template_guide/control_structures/

Reply:
Done